.github/workflows/ci.yml

```diff
@@ -65,6 +65,9 @@ jobs:
           - runner: ubuntu-24.04-arm
             platform: linux/arm64
     runs-on: ${{ matrix.runner }}
+    outputs:
+      digest-amd64: ${{ steps.build-amd64.outputs.digest }}
+      digest-arm64: ${{ steps.build-arm64.outputs.digest }}
```

Comment:
I'm not seeing how these outputs are getting populated, but I might be missing something?

Reply:
Missing id on the docker build step?

Reply:
> Missing id on the docker build step?

Yes I was. Just added it in. Thanks!